fastapi/onacut/routers/alerts.py

```diff
@@ -43,7 +46,38 @@ def get_alert(alert_id: int, db: Session = Depends(get_db)):
     responses={403: {"description": "Operation forbidden"}},
 )
 def create_alert(alert: AlertCreateSchema, db: Session = Depends(get_db)):
-    return {}
+    # check if the provided region exists in db
+    region = db.query(RegionModel).filter_by(id=alert.region_id).first()
+    if not region:
+        raise HTTPException(status_code=400, detail="Bad region's id!")
+
+    # check if the provided city exists in db
+    city = db.query(CityModel).filter_by(id=alert.city_id).first()
+    if not city:
+        raise HTTPException(status_code=400, detail="Bad city's id!")
+
+    # check if the provided districk exists in db
+    district = db.query(DistrictModel).filter_by(id=alert.district_id).first()
+    if not district:
+        raise HTTPException(status_code=400, detail="Bad district's id!")
+
+    # create the new Alert
+    db_alert = AlertModel()
+    db_alert.observations = alert.observations
+    db_alert.type = alert.type
+    db_alert.longitude = alert.longitude
+    db_alert.lattitude = alert.lattitude
+    db_alert.date = alert.date
+    db_alert.begin_time = alert.begin_time
+    db_alert.end_time = alert.end_time
+    db_alert.region_id = alert.region_id
+    db_alert.city_id = alert.city_id
+    db_alert.district_id = alert.district_id
```

Comment:
```suggestion
    db_alert = AlertModel(**alert.dict())
```

those objects have the same attribute name, what if we instantiate db_alert with the dict form of alert (which is a baseModel pydantic object)?

Reply:
ok i try it